docs/ecosystem/officials/index.md

```diff
@@ -16,6 +16,17 @@ Each project in the ZIO organization namespace has a _Stage Badge_ which indicat
 * **Concept** — The project is just an idea, development hasn't started yet.
 * **Deprecated** — The project is not maintained anymore, and we don't recommend its usage.
 
+## Highlights Projects
+
+Here is the list of some of the highlighted projects that can be used in everyday ZIO development:
```

Comment:
This makes it sound like other libraries are not suitable for everday ZIO development.